Colibri.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EjercicioDos
+{
+    internal class Colibri : Ave, Muerte
+    {
+        //creamos variable para guardar informacion del constructor        
+           ModeloAnimal guardarinfo;
+
+        //se sobre escribe metodo comer
+        public override void comer()
+        {
+            Console.WriteLine("el colibri come");
+        }
+        //se crea comportmiento diferente
+        public void volar()
+        {
+            Console.WriteLine("el colibri vuela");
+        }
+
+        //se reciben los parametros de muerte y se calcula cuanto vivio
+        public string morir(int nacimiento, int fallecimiento, string causa)
+        {
+            int calculo;
+            calculo = nacimiento - fallecimiento;
+            Console.WriteLine("La fecha de su muerte es: " + fallecimiento.ToString(), "vivio "+ calculo.ToString(), "años y la causa de la muerte es"+ causa);
+            
+            return Console.ReadLine();
```

Comment:
This does not make sense, you do not need to have this and return it. do this

```
	public string morir(int nacimiento, int fallecimiento, string causa)
        {
            int calculo;
            calculo = nacimiento - fallecimiento;
           return "La fecha de su muerte es: " + fallecimiento.ToString()+ " vivio "+ calculo.ToString()+ " años y la causa de la muerte es "+ causa
       
	}
```

and  the call it and print it from the program

Reply:
has been corrected